dataland-frontend/src/App.vue

```diff
@@ -65,12 +65,3 @@ export default defineComponent({
   },
 });
 </script>
-
-<style lang="scss">
```

Comment:
![image](https://user-images.githubusercontent.com/47553592/210356598-5e8f5a9b-b2c9-4569-b797-b35765b56511.png)
The text style in the buttons differs

Reply:
As discussed: please only take care about the font family. Otherwise, stick to the figma style

Reply:
Done. Buttons have font: "IBM Plex Sans"

Reply:
ok